app/views/curation_concerns/file_sets/media_display/_audio.html.erb

```diff
@@ -0,0 +1,40 @@
+<% if Rails.configuration.wowza_enabled
+  solr_document = file_set
+  solr_document = file_set.solr_document if (file_set.respond_to? :solr_document)
+
+  parent_id = parent.id if defined?(parent) && !parent.nil?
+  fedora_file = solr_document['digest_ssim'].first.split(':')[-1] if !solr_document['digest_ssim'].nil?
+
+  file_ext = 'mp3'
+  file_ext = solr_document['label_tesim'].first.split('.')[-1] if solr_document['label_tesim'].first
+
+  wowzaURL = grabWowzaURL(parent_id, file_set.id, fedora_file, file_ext, '127.0.0.1')
+puts "#{parent_id} #{file_set.id} #{fedora_file} #{file_ext} 127.0.0.1: #{wowzaURL}"
```

Comment:
Yes, we can use Rails.logger.info. But I think it's something I need to clean up unless we really want to log it here. What do you think?